### DISCUSSION TITLE
Interactive helper guide not showing up when I run ./generate.sh

### DISCUSSION THREAD

**Discussion by KrishivGubba** [airbytehq/airbyte#42934] [General]
### Topic

Airbyte low-code cdk

### Relevant information

I'm trying to develop a source connector using low-code cdk. I'm following the tutorial given by airbyte in this video:
https://www.youtube.com/watch?v=i7VSL2bDvmw
however, when I run ./generate.sh, the interactive helper guide does not show up? Instead the command builds a docker build called generator. Why is this so? Can someone help me out?

**Comment by natikgadzhi** [comment]
I'll try it out today! It's time for a little cleanup of its templates anyway.

[a comment by an automated account is not shown]
